### PR TITLE
[O11y][GCP] Add global filter for dashboards to improve performance related to infraobs packages

### PR DESCRIPTION
**Pull request by milan-elastic**
## Proposed commit message

Added global filter for below dashboards to improve performance,

- [Metrics GCP] Billing Overview
- [Metrics GCP] Load Balancing HTTPS Overview
- [Metrics GCP] Load Balancing L3 Overview 
- [Metrics GCP] Load Balancing TCP SSL Proxy Overview

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #10200 

## Screenshots

<details>
  <summary>[Metrics GCP] Billing Overview</summary>

| Before| After|
|:-:|:-:|
| gcp  metrics billing overview - before | gcp  metrics billing overview - after |
</details>

<details>
  <summary>[Metrics GCP] Load Balancing HTTPS Overview</summary>

| Before| After|
|:-:|:-:|
| gcp  load balancing https overview - before | gcp  load balancing https overview - after |
</details>

<details>
  <summary>[Metrics GCP] Load Balancing L3 Overview</summary>

| Before| After|
|:-:|:-:|
| gcp  load balancing l3 overview - before | gcp  load balancing l3 overview - after |
</details>

<details>
  <summary>[Metrics GCP] Load Balancing TCP SSL Proxy Overview</summary>

| Before| After|
|:-:|:-:|
| gcp  load balancing tcp ssl proxy overview - before | gcp  load balancing tcp ssl proxy overview - before |
</details>

### CODE REVIEW

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

File: packages/gcp/kibana/dashboard/gcp-8f9c6cc0-909d-11ea-8180-7b0dacd9df87.json [hunk 1]
```diff
@@ -804,20 +809,23 @@
...
     },
-    "coreMigrationVersion": "8.7.1",
-    "created_at": "2023-08-18T06:53:59.299Z",
+    "coreMigrationVersion": "8.8.0",
+    "created_at": "2024-04-22T11:50:08.027Z",
     "id": "gcp-8f9c6cc0-909d-11ea-8180-7b0dacd9df87",
-    "migrationVersion": {
-        "dashboard": "8.7.0"
-    },
+    "managed": false,
```
Comment (gpop63) [inline packages/gcp/kibana/dashboard/gcp-8f9c6cc0-909d-11ea-8180-7b0dacd9df87.json:822]: Shouldn't this be `true` for all dashboards?

Comment (harnish-elastic) [reply packages/gcp/kibana/dashboard/gcp-8f9c6cc0-909d-11ea-8180-7b0dacd9df87.json:822]: We have exported the dashboards using the `elastic-package export dashboards` command and found this difference.  We have also tested after exporting the dashboard and found no difference. I think we shouldn't change this values manually! WDYT?

Comment (gpop63) [reply packages/gcp/kibana/dashboard/gcp-8f9c6cc0-909d-11ea-8180-7b0dacd9df87.json:822]: I noticed that some other dashboards have `managed: false` as well. It should be fine.

**Review by shmsr — APPROVED** [review]

**Pull request merged**